.devops/soak-tests-pipeline.yml

```diff
@@ -32,7 +32,11 @@ parameters:
       - getNoticeReceipt
       - getPaymentNotices
       - getZendeskAssistanceToken
-      - getTransactions
+      - runAll
+      - runAllAuth
+      - runAllNotice
+      - runAllPaymentNotice
+      - runAllTokenAssistance

```

Comment:
if we rearrange this list in order to have:
* runAll as first
* runAll-<domain> before all domain related tests?